### DISCUSSION TITLE
Zasób /sale/categories/{}/product-parameters zwracanie wartości ambigousID dla parametru

### DISCUSSION THREAD

**Discussion by szymoki** [allegro/allegro-api#6253] [Suggestions - rejected]
Zasób /sale/categories/{categoryId}/product-parameters - brakuje w nim dla parametru słownikowego, informacji która wybrana wartość jest niejednoznaczna (ambigousId), jak to jest w zasobie /sale/categories/{categoryId}/parameters

W sytuacji dodawania propozycji produktu przez API w wybieraniu wartości słownikowej "inna", nie ma tego jak zweryfikować czy w tym parametrze powinna być podana odrębna wartość tesktowa czy nie.

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
